File: src/main/java/no/nav/fo/veilarb/dokument/domain/Adresse.java
package no.nav.fo.veilarb.dokument.domain;


import lombok.Value;
import lombok.experimental.Accessors;

@Value
@Accessors(fluent = true)
public class Adresse {
    String adresselinje1;
    String adresselinje2;
    String adresselinje3;
    String land;
    String postnummer;
    String poststed;
}

File: src/main/java/no/nav/fo/veilarb/dokument/domain/Adresse.kt
package no.nav.fo.veilarb.dokument.domain

data class Adresse(
        val adresselinje1: String,
        val adresselinje2: String? = null,
        val adresselinje3: String? = null,
        val postnummer: String,
        val poststed: String,
)
